pkg/agent/openflow/pipeline.go

```diff
@@ -41,38 +41,127 @@ import (
+	PipelineClassifierTable = newTable("PipelineClassifier", binding.StageAll, binding.PipelineAll)
+
+	//      _   _   _             _   _               _
+	//     / \ | |_| |_ ___ _ __ | |_(_) ___  _ __   | |
+	//    / _ \| __| __/ _ \ '_ \| __| |/ _ \| '_ \  | |
+	//   / ___ \ |_| ||  __/ | | | |_| | (_) | | | | |_|
+	//  /_/   \_\__|\__\___|_| |_|\__|_|\___/|_| |_| (_)
+	//
+	// Before adding a new table here, please read the following instruction carefully.
+	// - Double confirm the necessity of adding a new table, and consider how to reuse an existing table to implement
+	//   alternative functionality.
+	// - Choose a good name. A good name is very helpful for users to understand the function of the table.
+	// - Choose a stage. All tables in a stage are intended to implement a functional module in pipelines. All stages are
+	//   defined in file pkg/ovs/openflow/interfaces.go. If you want to add a new stage, please discuss with maintainers
+	//   or developers of Antrea.
+	// - Choose a target pipeline. All pipelines are defined in file pkg/ovs/openflow/interfaces.go. If you want to add a
+	//   new pipeline, please discuss with maintainers or developers of Antrea.
+	// - The most important thing is to choose the right place to declare the new table. For example:
+	//     * If you want to add a table called `DummyTable` between `SpoofGuardTable` and `IPv6Table` in pipeline for IP,
+	//       then the table should be declared as follows:
+	//       ```go
+	//          SpoofGuardTable  = newTable("SpoofGuard", binding.ValidationStage, binding.PipelineIP)
+	//          DummyTable       = newTable("Dummy", binding.ValidationStage, binding.PipelineIP)
+	//	        IPv6Table        = newTable("IPv6", binding.ValidationStage, binding.PipelineIP)
+	//	        IPClassifierTable = newTable("IPClassifier", binding.ValidationStage, binding.PipelineIP)
+	//       ```
+	//      * If you want to add a table called `DummyTable` just before `ARPResponderTable` in pipeline for ARP, then the
+	//        table should be declared as follows:
+	//       ```go
+	//          DummyTable        = newTable("Dummy", binding.OutputStage, binding.PipelineARP)
+	//          ARPResponderTable = newTable("ARPResponder", binding.OutputStage, binding.PipelineARP)
+	//       ```
+	//       * If you want to add a table called `DummyTable` just after `ConntrackStateTable` in pipeline for ARP, then
+	//         the table should be declared as follows:
+	//       ```go
+	//          SNATConntrackTable  = newTable("SNATConntrackZone", binding.ConntrackStateStage, binding.PipelineIP)
+	//          ConntrackTable      = newTable("ConntrackZone", binding.ConntrackStateStage, binding.PipelineIP)
+	//          ConntrackStateTable = newTable("ConntrackState", binding.ConntrackStateStage, binding.PipelineIP)
+	//	        DummyTable          = newTable("Dummy", binding.ConntrackStateStage, binding.PipelineIP)
+	//       ```
```

Comment:
@jianjuns Sorry for missed the question.

> We should add a comment saying the order of declaring the tables decides the order of the tables in the stage.

I added an example here to explain how to add new table and arrange it. Do you think we still need to add the comment saying `the order of declaring the tables decides the order of the tables in the stage` before the definitions of tables in a stage?